more_itertools/more.py

```diff
@@ -128,6 +128,14 @@ def chunked(iterable, n):
         >>> list(chunked([1, 2, 3, 4, 5, 6, 7, 8], 3))
         [[1, 2, 3], [4, 5, 6], [7, 8]]
 
+    If this is the case and *strict* is True, then :func:`chunked` will
+    raise an exception.
+
+        list(chunked([1, 2, 3, 4, 5, 6, 7, 8], 3, strict=True))
```

Comment:
I think this needs `>>>` in front like the other code blocks. Also possibly the `IGNORE_EXCEPTION_DETAIL` [directive].